examples/are-you-alive/deploy/README.md

```diff
@@ -0,0 +1,5 @@
+# Kubernetes Deployment
+
+Configuration for deploying `are-you-alive` on Kubernetes.
+
+Also deploys Prometheus and Alertmanager.
```

Comment:
are we really deploying AlertManager? I don't see it.

Reply:
Ah, thanks for calling this one out.  I'll just delete this directory.